index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/138cf4be1d.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <title>Akpuru Solomon Website</title>
+</head>
+<body>
+  <div id="container">
+   <header class="header">
+      <div class="topnav" id="myTopnav">
+         <a id="bars" onclick="" href="#"><i class="fa fa-bars"></i></a>
+        <a id="mess" href="#"><img src="./image/Union.png" alt="Message Icon" /></a>
+        <a id="logo" href="#">Akpuru Solomon</a>
+        <a href="#works" class="active">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+    
+      </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove the unnecessary empty code lines in your code base to make it more liable and for best practices.